src/zjs_script.c

```diff
@@ -46,6 +46,7 @@ uint8_t zjs_read_script(char* name, const char** script, uint32_t* length)
             return 1;
         }
 
+        s[size] = '\0';
```

Comment:
This one writes past the end of the array too. You need to malloc size+1 to do this.